projects/ng-gallery-demo/src/app/shared/badges/badges.component.html

```diff
@@ -14,13 +14,13 @@
   <img data-canonical-src="https://travis-ci.org/MurhafSousli/ngx-gallery"
        src="https://travis-ci.org/MurhafSousli/ngx-gallery.svg?branch=master">
 </a>
-<a href="https://www.npmjs.com/package/@ngx-gallery/core">
-  <img data-canonical-src="https://www.npmjs.com/package/@ngx-gallery/core"
-       src="https://img.shields.io/npm/dt/@ngx-gallery/core.svg?maxAge=2592000?style=plastic">
+<a href="https://www.npmjs.com/package/ngx-gallery">
```

Comment:
@MurhafSousli isn't the `ngx-gallery` package a different one?

Reply:
Oh yea, thanks for noticing! will need to fix that